scenes/main_menu/main_menu_level.gd

```diff
@@ -2,20 +2,29 @@
 extends "../scene_base.gd"
 
 func _update_demo_positions() -> void:
-	var count = $Demos.get_child_count()
+	var count = 0#$Demos.get_child_count()
+	var vis_chd := []
```

Comment:
I'm not a big fan of shorthand unless names get really long, I would rename this to `visible_children`.

Reply:
Ok, I gona do it, but I don't think phone github will allow me so probably I gona sit to it in 8H